### PR TITLE
Option

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Comment by matklad** [comment]
This is for Rust 1.22, to be released on thursday.

File: src/test/kotlin/org/rust/lang/core/resolve/RsStdlibResolveTest.kt [hunk 1]
```diff
@@ -433,6 +435,22 @@ class RsStdlibResolveTest : RsResolveTestBase() {
         }
     """)
 
+    fun `test ? operator with option`() {
```
Comment (vlad20012) [inline src/test/kotlin/org/rust/lang/core/resolve/RsStdlibResolveTest.kt]: =

**Comment by matklad** [comment]
bors r=vlad20012

**Comment by matklad** [comment]
bors retry

**Comment by matklad** [comment]
I should have named this method `NUL`...

**Pull request merged**